src/clr-angular/data/tree-view/models/expanded-state.enum.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export enum ClrExpandedState {
+  EXPANDED = 'expanded',
+  COLLAPSED = 'collapsed',
+}
```

Comment:
Is this needed because `childNodesState` animation is using `expanded` and `'collapsed` as its state names?

Reply:
That was the intention, but I guess it's not mandatory. It would be nice to have textual states we can use directly in the tests, I think.

Reply:
@Shijir I don't think we have other option if we want to keep the child nodes visible before the animation is over. 
If the team agrees synchronicity is more important - I'm okay to drop this change entirely.

Reply:
I double checked my local tests and I am able to write unit tests without worrying about using fakeAsync or tick as long as the test module uses the Noop Animations module. Still going to research more into this since the stepper has to use the same API.